js/packages/phoenix-client/src/types/datasets.ts

```diff
@@ -0,0 +1,21 @@
+import { Node } from "./core";
+
+/**
+ * An example is a record to feed into an AI task
+ */
+export interface Example extends Node {
+  id: string;
+  updatedAt: Date;
```

Comment:
should this be nullable? thinking from perspective of ad hoc datasets

Reply:
Yeah we can relax as we go

Reply:
I'm just matching the python right now